doc/source/user_guide/copy_on_write.rst

```diff
@@ -52,6 +52,74 @@ it explicitly disallows this. With CoW enabled, ``df`` is unchanged:
 The following sections will explain what this means and how it impacts existing
 applications.
 
+Migrating to Copy-on-Write
+--------------------------
```

Comment:
I wonder if we should move the migration guide further in the file, as right now this section assumes somewhat knowledge about what CoW is, but that is only explained "Description" section that now comes afterwards

Reply:
I had similar thoughts, but I wanted to put it as prominent as possible